code/obj/item/device/pda2/pda2.dm

```diff
@@ -49,7 +49,7 @@
 	var/setup_default_module = /obj/item/device/pda_module/flashlight //Module to have installed on spawn.
 	var/mailgroups = list("staff",MGD_PARTY) //What default mail groups the PDA is part of.
 	var/muted_mailgroups = list() //What mail groups should the PDA ignore?
-	var/reserved_mailgroups = list(MGD_COMMAND,MGD_SECURITY,MGD_SCIENCE,"ai","sillicon",MGD_MEDRESEACH,MGD_MEDBAY ,MGD_CARGO,"janitor","chaplain","engineer","mining",MGD_KITCHEN,"mechanic",MGD_BOTANY) //Job-specific mailgroups that cannot be joined or left
+	var/reserved_mailgroups = list(MGD_COMMAND,MGD_SECURITY,MGD_SCIENCE,"ai","sillicon",MGD_MEDRESEACH,MGD_MEDBAY ,MGD_CARGO,"janitor","chaplain","engineer","mining",MGD_KITCHEN,"mechanic",MGD_BOTANY,MGD_SPIRITUALAFFAIRS) //Job-specific mailgroups that cannot be joined or left
```

Comment:
why is there chaplain and your new addition here

Reply:
No good reason: I wasn't paying full attention.
Replaced "chaplain" in other appropriate locations now that I am paying attention.